### PR TITLE
Red V4 Release

### PR DESCRIPTION
**Pull request by Drapersniper**
The moment you all have been waiting for Red V4 is here.

Storage Backend
Don't like Config managing your database for you? Take back control with DataIO!

Music Playback
We've got less commands in audio now! Back to the basics.
No traffic egress used on your vps!
100% less memory used than v3's audio!

Fun Trivia games for your server
Edit your trivia files easier with the new and improved PLAIN TEXT FORMAT (no yaml formatting to learn)

Stream Alerts
Do they work? We don't know! But that's better than V3 eating your entire YouTube API quota!

3rd Party Cogs
Approved cog creator? Unapproved cog creator? Who needs those labels! Just hit up GitHub for over 1400 cogs that may or may not crash your bot.

* disclaimer: V4 should not be used for any Discord bot installation that actually needs to work and we will not be help accountable for you using this version of V4

### CODE REVIEW

**Review by Jackenmen — DISMISSED** [review]
LGTM, that's the moment we've been waiting for!

**Comment by aikaterna** [comment]
Man, who knew. No Audio is so much better. Approved

**Review by PredaaA — CHANGES_REQUESTED** [review]
Can you not Draperize this PR too? Resolve those conflicts and it'll be ready to go.
74JZI-draperizedhonk

**Review by NeuroAssassin — DISMISSED** [review]
I actually like it how it is, I think the draperization makes it better

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -47,136 +27,49 @@ Unsure of how to get started contributing to Red? Please take a look at the Issu
...
-
-### 4.4 Make
-You may have noticed we have a `Makefile` and a `make.bat` in the top-level directory. For now, you can do a few things with them:
-1. `make reformat`: Reformat all python files in the project with Black
-2. `make stylecheck`: Check if any `.py` files in the project need reformatting
-3. `make newenv`: Set up a new virtual environment in the `.venv` subdirectory, and install Red and its dependencies. If one already exists, it is cleared out and replaced.
-4. `make syncenv`: Sync your environment with Red's latest dependencies.
+# Getting Started
+### Testing
+We've recently started adding unit-testing into Red. All current tests can be found in the `tests/` directory at the root level of the repository. You will need `py.test` installed in order to run them (which is already in `requirement.txt`). Tests can be run by simply calling `pytest` once you've `cd`'d into the Red repository folder.
```
Comment (Flame442) [inline .github/CONTRIBUTING.md:32]: Wouldn't this be `requirements.txt`?

**Review by Flame442 — CHANGES_REQUESTED** [review]
This looks like a really good change, thanks for contributing! Just need to sort out this one little issue then we can get this merged.

**Comment by Stonedestroyer** [comment]
I tried to run this and I get a websocket error, it seems audio is broken?

**Comment by Drapersniper** [comment]
> I tried to run this and I get a websocket error, it seems audio is broken?

No it’s a hidden feature

**Comment by fixator10** [comment]
But can it run Minecraft?

**Review by Jackenmen — CHANGES_REQUESTED** [review]
I'm -1 on this, this would require us to maintain over 10k lines of code instead of maybe 250.

**Comment by PredaaA** [comment]
Are you excepting to put docs on audio cog?

**Comment by Drapersniper** [comment]
People aren't Ready for the awesomeness of a V4 release

**Pull request closed without merging**